### PR TITLE
adds updates for 6_getAllArticleComments

### PR DESCRIPTION
**Pull request by ericnabofa**

### CODE REVIEW

File: Components/Comments.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        getArticleComments(articleId).then((commentsFromApi) => {
+            setComments(commentsFromApi)
+        })
+    }, [articleId])
+
+
+return (
+    <section>
+        <h2>Comments</h2>
+        <ol>
```
Comment (itskatherine) [inline Components/Comments.jsx:19]: Though it's great you've rendered a list of comments, I would maybe reconsider a **numbered** list of comments 🔢 
Look at sites like Youtube or Reddit for inspiration about what info is useful to display, things like the username, when it was posted for example

File: Components/Comments.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react"
+import { getArticleComments } from "../src/utils/api"
+
+
+const Comments = ({articleId}) => {
+
+    const [comments, setComments] = useState([])
+
```
Comment (itskatherine) [inline Components/Comments.jsx:8]: Can you please add a loading state for the comments and add a loading message ⌛

File: Components/Comments.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react"
+import { getArticleComments } from "../src/utils/api"
+
+
+const Comments = ({articleId}) => {
```
Comment (itskatherine) [inline Components/Comments.jsx:5]: Nice use of destructuring ✅

File: src/utils/api.js [hunk 4]
```diff
@@ -14,7 +14,14 @@ export const getSingleArticle = (article_id) => {
...
         return data.article
     })
 }
+
+export const getArticleComments = (article_id) => {
+    return api
+    .get(`/api/articles/${article_id}/comments`)
+    .then(({data}) => {
+        return data.comments
+    })
```
Comment (itskatherine) [inline src/utils/api.js:26]: Nice separation of your API functions in this file

**Review by itskatherine — COMMENTED** [review]
Hi Eric, nice work on adding commentd but a few things to add, and maybe consider the layout in info displayed in your comments a bit more
But once you've addressed my comments you're good to merge 👍

File: Components/Comments.jsx (hunk 1 above)
Comment (ericnabofa) [reply Components/Comments.jsx:19]: okay, i will effect this now. thank you

File: Components/Comments.jsx (hunk 2 above)
Comment (ericnabofa) [reply Components/Comments.jsx:8]: yup, thank you

**Comment by ericnabofa** [comment]
Hello katherine, i have already effected the changes. could you please also help me feedback on the voting on an article before end of work today so i can focus on post and deleting comments as against tomorrow pls

thank you

**Pull request merged**